composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "winnersoftlab/dbal-vertica-driver",
+    "name": "jnmik/dbal-vertica-driver",
```

Comment:
@JnMik, thank your for your request. I will accept it, but can't do it with your vendor name. Please, can you revert it? Because if I accept it now, projects already uses this packeges will fail on update.